fuzz/fuzz_targets/fuzz_cksum.rs

```diff
@@ -0,0 +1,159 @@
+// This file is part of the uutils coreutils package.
+//
+// For the full copyright and license information, please view the LICENSE
+// file that was distributed with this source code.
+// spell-checker:ignore chdir
+
+#![no_main]
+use libfuzzer_sys::fuzz_target;
+use std::ffi::OsString;
+use uu_cksum::uumain;
+mod fuzz_common;
+use crate::fuzz_common::{
+    compare_result, generate_and_run_uumain, generate_random_file, generate_random_string,
+    run_gnu_cmd, CommandResult,
+};
+use rand::Rng;
+use std::env::temp_dir;
+use std::fs::{self, File};
+use std::io::Write;
+use std::process::Command;
+
+static CMD_PATH: &str = "/usr/bin/cksum";
+
+fn generate_cksum_args() -> Vec<String> {
+    let mut rng = rand::thread_rng();
+    let mut args = Vec::new();
+
+    let digests = [
+        "sysv", "bsd", "crc", "md5", "sha1", "sha224", "sha256", "sha384", "sha512", "blake2b",
+        "sm3",
+    ];
+    let digest_opts = ["--base64", "--raw", "--tag", "--untagged"];
```

Comment:
How about adding `--text` and `--binary` in the mix? That could result in interesting sums-files which might reveal a difference in interpretation.